# enforce uri named properties include uri format (properties-format-uri)

Validates that any properties with `url` or `uri` in the name include `{"format": "uri"}`.

## Examples of *correct* usage

```json
{
  "TODO"
}
```

## Examples of **incorrect** usage

```json
{
  "TODO"
}
```

```json
{
  "TODO"
}
```
