# enforce all properties' keys are `snake_case` style (properties-style)

Validates that all properties' keys are `snake_case` style.

## Examples of *correct* usage

```json
{
  "TODO"
}
```

## Examples of **incorrect** usage

```json
{
  "TODO"
}
```

```json
{
  "TODO"
}
```
