# enforce country properties are named `country_code` or end with `_country_code` (properties-country-code)

Validates that properties with the word `country` are named `country_code`, or end with `_country_code`.

## Examples of *correct* usage

```
{
  TODO
}
```

## Examples of **incorrect** usage

```
{
  TODO
}
```

```
{
  TODO
}
```
