# enforce all properties' keys are `snake_case` style (properties-style)

Validates that all properties' keys are `snake_case` style.

## Examples of *correct* usage

```
{
  TODO
}
```

## Examples of **incorrect** usage

```
{
  TODO
}
```

```
{
  TODO
}
```
