# enforce uri named properties include uri format (properties-format-uri)

Validates that any properties with `url` or `uri` in the name include `{"format": "uri"}`.

## Examples of *correct* usage

```
{
  TODO
}
```

## Examples of **incorrect** usage

```
{
  TODO
}
```

```
{
  TODO
}
```
